server/src/main/java/io/deephaven/server/table/ExportedTableUpdateListener.java

```diff
@@ -174,7 +187,7 @@ private synchronized void onNewTableExport(final Ticket ticket, final int export
                 sendUpdateMessage(ticket, -1, Exceptions.statusRuntimeException(Code.FAILED_PRECONDITION,
                         "Exported Table Already Failed"));
             } else {
-                sendUpdateMessage(ticket, -1, GrpcUtil.securelyWrapError(log, err, Code.FAILED_PRECONDITION));
+                sendUpdateMessage(ticket, -1, errorTransformer.transform(err));
```

Comment:
We lost the status code here. Seems bad.

Reply:
Colin says that ILLEGAL_ARGUMENT is a slight preference for him anyway. Technically these errors aren't the RPC stream themselves, but being embedded in the ETUM stream. I'd prefer that grpc status code is not part of the transform API anyway.

We're going to leave this one as is.